vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -3246,6 +3249,9 @@ public void sort(List<GridSortOrder<T>> order) {
         if (order == null) {
             order = Collections.emptyList();
         }
+        if (!isMultiSort() && order.size() > 1) {
+            order = order.subList(0, 1);
```

Comment:
Should a warning also be shown as suggested in https://github.com/vaadin/flow-components/issues/3155#issuecomment-1135755378?

Reply:
Thanks, added a warning.